man/org.freedesktop.systemd1.xml

```diff
@@ -1403,7 +1403,8 @@ node /org/freedesktop/systemd1 {
       <function>DumpByFileDescriptor()</function>/<function>DumpUnitsMatchingPatternsByFileDescriptor()</function>
       are usually the preferred interface, since it ensures the data can be passed reliably from the service
       manager to the client. Note though that they cannot work when communicating with the service manager
-      remotely, as file descriptors are strictly local to a system.</para>
+      remotely, as file descriptors are strictly local to a system. All the <function>Dump*()</function>
+      methods are rate limited to 10 calls in 10 minutes for unprivileged users.</para>
```

Comment:
i wouldn't document the precise params here. That way you make it API. Just say it's rate limited, without specifying the rate.

Reply:
seems weird to have a limit and not document it, but sure